test/http.spec.js

```diff
@@ -0,0 +1,29 @@
+import nock from 'nock';
+import path from 'path';
+import downloadFile from '../src/download-file';
+import fs from 'fs';
+import expect from 'expect';
+
+nock('http://local.foo.com')
```

Comment:
@alejandronanez you mean like creating generic functions that setup a server in a very specific condition? I think it makes sense if there are going to be a few, but we should see how many.